examples/psbt.rs

```diff
@@ -0,0 +1,33 @@
+extern crate bitcoin;
+
+use std::{env, process};
+
+use bitcoin::util::psbt::PartiallySignedTransaction;
+use bitcoin::consensus::encode::deserialize;
+
+fn main() {
+
+  let args: Vec<String> = env::args().collect();
+  if args.len() < 2 {
+      eprintln!("not enough arguments. usage: cargo run --example psbt <PSBT>");
+      process::exit(1);
+  }
+  let decoded_psbt = match base64::decode(&args[1]){
+    Ok(psbt)=> psbt,
+    Err(_)=>{
+      eprintln!("Error decoding psbt from base64");
+      process::exit(1);
+    }
+  };
+
+  let psbt_struct: PartiallySignedTransaction = match deserialize(&decoded_psbt){
+    Ok(psbt)=>psbt,
+    Err(_)=>{
+      eprintln!("Error deserialising psbt");
+      process::exit(1);
+    }
+  };
+
+  println!("{:#?}", &psbt_struct);
+
```

Comment:
I don't rekon we need this whitespace, or that at the start of `main()`. Damn this feels like reviewing C code ...

Reply:
:p sorted.